### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by lucyleeow**
I've made some changes to the 'getting started' page to make it make it more friendly for new users and more cohesive.

Happy to amend if anything is incorrect/misleading or if too much detail.

### CODE REVIEW

**Comment by massich** [comment]
I think this is a legit error https://ci.appveyor.com/project/sphinx-gallery/sphinx-gallery/builds/26920140/job/mcd4da30aghwbreo#L667

File: doc/getting_started.rst [hunk 1]
```diff
@@ -8,10 +8,11 @@ Creating a basic Gallery
...
 
 This section describes how to set up a basic gallery for your examples
-using ``sphinx-gallery``. You'll do the following.
+using the Sphinx extension Sphinx-Gallery, which will do the following:
 
-* Automatically generate sphinx rST for your examples, along with Jupyter
-  notebooks of each that users can download.
+* Automatically generate Sphinx rST for your example ``.py`` files, along with
+  ``.ipynb`` (Jupyter notebook) and ``.py`` files of each example that users 
+  can download.
```
Comment (massich) [inline doc/getting_started.rst]: * Automatically generate Sphinx rST out of your ``.py`` example files. 
  The rendering of the resulting rST will provide the users with 
  ``.ipynb`` (Jupyter notebook) and ``.py`` files of each example to be downloaded.

File: doc/getting_started.rst [hunk 2]
```diff
@@ -41,57 +43,63 @@ Let's say your Python project looks like this:
...
+      ``README.txt`` or ``README.rst`` file as well.
 
 Configure and use sphinx-gallery
 --------------------------------
 
 After Sphinx-Gallery is installed, we must enable and configure it to build
 with Sphinx.
 
-First, enable Sphinx-Gallery in the Sphinx ``conf.py`` file with::
+First, enable Sphinx-Gallery in the Sphinx ``doc/conf.py`` file with::
```
Comment (massich) [inline doc/getting_started.rst]: +1 (but this might be really personal taste)

File: doc/getting_started.rst [hunk 3]
```diff
@@ -102,41 +110,76 @@ This loads Sphinx-Gallery as one of your extensions, the ellipsis
...
+* ``sg_execution_times.rst`` - execution time of all example ``.py`` files, 
+  summarised in table format (`original pull request on GitHub
+  <https://github.com/sphinx-gallery/sphinx-gallery/pull/348>`_).
+* ``images`` - directory containing images produced during execution of the 
+  example ``.py`` files (more details in :ref:`image_scrapers`) and thumbnail
+  images for the gallery.
+* A directory for each sub-directory in 'example_dirs'. Within each directory
+  will be the above and below listed files for that 'sub-gallery'.
+
+Additionally for EACH ``.py`` file, a file with the following suffix is
```
Comment (massich) [inline doc/getting_started.rst]: make it bold rather than capital letters.

File: doc/getting_started.rst [hunk 4]
```diff
@@ -102,41 +110,76 @@ This loads Sphinx-Gallery as one of your extensions, the ellipsis
...
     $ make html
 
-This will start the build of your complete documentation including the examples
-gallery. Once a build is completed, all your examples outputs are cached.
+This will start the build of your complete documentation. Both 
+the Sphinx-Gallery output files described above and 
+the Sphinx output HTML (or another output format depending on the 
+`Sphinx 'builder'
+<https://www.sphinx-doc.org/en/master/man/sphinx-build.html>`_ selected) will
+be generated. Once a build is completed, all your examples' outputs are cached.
```
Comment (massich) [inline doc/getting_started.rst]: I would use `all the outputs from your examples will be cached.` I find `'` hard for none English speakers.

**Comment by choldgraf** [comment]
(also - thanks so much @lucyleeow for the contribution, I think this is quite helpful!)

**Comment by larsoner** [comment]
Pushed a commit to add bullet points and adjust capitalization, otherwise LGTM. Will merge once CIs come back happy and the CircleCI rendering looks good, thanks @lucyleeow !

**Comment by lucyleeow** [comment]
Thanks for fixing the formatting @larsoner. Still not sure why the 2nd commit failed CI on Windows but...

**Comment by larsoner** [comment]
AppVeyor builds have been a little bit flaky

**Pull request merged**